pkg/cluster/k8sres.go

```diff
@@ -226,6 +226,25 @@ PATRONI_INITDB_PARAMS:
 	return string(result)
 }
 
+func (c *Cluster) nodeAffinity() *v1.Affinity {
+	matchExpressions := make([]v1.NodeSelectorRequirement, 0)
+	for k, v := range c.OpConfig.CordonedNodeLabels {
```

Comment:
Is the naming wrong? We want to schedule on nodes with "ready" thus the not cordoned nodes.